src/EnergyPlus/SimulationManager.cc

```diff
@@ -281,6 +281,7 @@ namespace SimulationManager {
         using SizingManager::ManageSizing;
         using SystemReports::CreateEnergyReportStructure;
         using SystemReports::ReportAirLoopConnections;
+        using SystemReports::AllocateAndSetUpVentReports;
```

Comment:
@LipingWang Minor detail. We're trying to move away from `using` statements and be explicit when actually using the function or variable. So, when you add the unit test, please delete this line.